Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM node:0.12-onbuild
```

Comment:
I'd prefer if we run without the `-onbuild` tag - `node:0.12`, or even `node:4` (but wait until Monday for critical HTTP DoS fix).

@astorije What the `-onbuild` tag indicates is that it will automatically do this;

```
WORKDIR /usr/src/app
ONBUILD COPY package.json /usr/src/app
ONBUILD RUN npm install
ONBUILD COPY . /usr/src/app

CMD ["npm", "start"]
```

What this essentially means is that it will take care of moving source files into the image, and making sure dependencies is installed when running the image. It also specifies a default command (`$ npm start`) to be executed when running the image (this will be overridden by our `ENTRYPOINT`).